OasisDEX/oasis-borrow
# Updated folder structure, unified naming convention

# [Update automation file structure](https://app.shortcut.com/oazo-apps/story/5777/update-files-structure-in-protection-and-optimization-directories)

# [Unify automation naming convention](https://app.shortcut.com/oazo-apps/story/5776/rename-bs-related-methods-with-buy-sell-prefix)


<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- updated folder structure and unified naming convention in scope of automation
  
## How to test 🧪
  <Please explain how to test your changes>

- do round of test to make sure that everything works correctly (this PR is only about refactor so nothing should change from app point of view)
